contents/handbook/company/management.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Management at PostHog
+sidebar: Handbook
+showTitle: true
+---
+
+As we grow, we'll increase the number of managers at PostHog. Here's what a manager at PostHog looks like.
+
+## Defining the role of manager
+
+A manager at PostHog has these tasks
+1. Making sure their direct reports are happy and productive
+1. Setting the right context for direct reports to do their job
+
+That's it.
+
+A manager at PostHog is _not_ responsible for:
+1. Setting compensation (we have transparent compensation)
```

Comment:
Who is signing compensation - is that James and Tim? While the calculation is transparent, someone ultimately decides you are at Step X or we are benchmarking you against role Y as inputs.

Reply:
Good point. I'd suggest the framework is centralized, whereas the "you are at X" should sit with the manager because they'll know how the person is getting on much better.

Reply:
Yeah that's a fair point. I think even up until 40-50 people (ie end of year) we'll still want to make even those decisions centrally. After that we could add something like 'decide what step you're on' but i don't think that's relevant right now